src/components/Articles.jsx

```diff
@@ -0,0 +1,72 @@
+import { useContext, useEffect } from "react";
+import { ArticleContext } from "../contexts/Article";
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+} from "@mui/material";
+import { getArticles } from "../api";
+
+const Articles = () => {
+  const { articles, setArticles } = useContext(ArticleContext);
+  console.log(articles);
+
+  useEffect(() => {
+    getArticles().then((articlesData) => {
+      setArticles(articlesData.articles);
+    });
+  }, []);
+
+  return (
+    <ul>
+      {articles.map((article) => (
+        <li key={article.article_id}>
+          <Card sx={{ minWidth: 275 }}>
```

Comment:
There's quite a lot of functionality in here on each element of this map, you may want to extract this out into an articleCard component for further abstraction.